redaxo/src/addons/debug/boot.php

```diff
@@ -21,7 +21,7 @@
     }
 
     // prepend backend folder
-    $apiUrl = rex_path::basename(rex_path::backend()).'/'.rex_debug_clockwork::getClockworkApiUrl();
+    $apiUrl = dirname($_SERVER['REQUEST_URI']).'/'.rex_debug_clockwork::getClockworkApiUrl();
```

Comment:
ich weiß nicht, ob es noch andere cases in REDAXO gibt, aber ggf. eine Methode im core, die einem die richtige frontend/backend url raus gibt, wäre hilfreich IMHO. 🤷🏼‍♂️

Reply:
hab ich mich auch gefragt 😅